src/index.js

```diff
@@ -1,24 +1,24 @@
 import createjs from 'createjs';
 import $ from 'jquery';
 import AccessibilityModule from 'createjs-accessibility';
-import AppWindow from './widgets/AppWindow.js';
+import AppWindow from './widgets/AppWindow';
 
 function init() {
   const stage = new createjs.Stage('stage');
   createjs.Touch.enable(stage, true, false);
   createjs.Ticker.framerate = 24;
 
   // todo: delete
-  window.stage = stage;
+  // window.stage = stage;
```

Comment:
How about deleting lines 11-13 instead of commenting this out?  It was only put in for debug purposes early in the project.

Reply:
Done